app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentTv.kt

```diff
@@ -575,6 +575,7 @@ class ResultFragmentTv : Fragment() {
         observeNullable(viewModel.movie) { data ->
             binding?.apply {
                 resultPlayMovie.isVisible = data is Resource.Success
+                resultPlaySeries.isVisible = data == null
```

Comment:
Series button visible on null movie? why

Reply:
I am hidding it for movie, as it takes the focus for movie